### PR TITLE
Feature/ayokunle/add main screen

### PR DESCRIPTION
**Pull request by mrayor**
- Added Main Screen Navigations

- Integrated Maps

### CODE REVIEW

**Comment by mrayor** [comment]
> Why does the home screen contain `Home Screen` as text
It doesnt, it has the `<Map />` component

File: src/components/screens/mainScreens/MapIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { Ionicons } from "@expo/vector-icons";
+import {IONICONS} from '../constants'
+
+const {MD_PIN} = IONICONS
```
Comment (parkerthegeniuschild) [inline src/components/screens/mainScreens/MapIcon.jsx]: {} styling is not consistent, kindly use linting to fix

File: src/components/screens/mainScreens/HomeScreen.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import {Map} from "../../home/components";
+
+const HomeScreen = () => {
+  return <Map />;
+};
+export { HomeScreen };
```
Comment (parkerthegeniuschild) [inline src/components/screens/mainScreens/HomeScreen.jsx]: {} styling is not consistent, kindly use linting to fix

File: src/components/home/components/Map.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import { MapStyles } from "./styles";
+
+const { container, mapStyle } = MapStyles;
+
+const Map = () => {
+  const [region, setRegion] = useState({
+    latitude: 6.465422,
+    longitude: 3.406448,
+    latitudeDelta: 0.95,
+    longitudeDelta: 0.45
```
Comment (parkerthegeniuschild) [inline src/components/home/components/Map.jsx]: You could throw this object in a constant. I presume it's your default view when the map loads

File: .gitignore [hunk 4]
```diff
@@ -15,4 +15,7 @@ web-report/
 # Yarn Integrity file
 .yarn-integrity
 /yarn.lock
-package-lock.json
\ No newline at end of file
+package-lock.json
+
+#app.json
+/app.json
```
Comment (parkerthegeniuschild) [inline .gitignore]: insert new line  . .  EOL

**Pull request merged**